### DISCUSSION TITLE
MacBook Pro 14 M2 Pro has started shutting down when the battery level drops below 40%

### DISCUSSION THREAD

**Discussion by AndriiAlerhush** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1197] [Q&A]
Hello,
I've noticed that my laptop (MacBook Pro 14 M2 Pro) has started shutting down when the battery level drops below 40%. It won't turn back on until I connect the power cable. Because of this, I completely removed the application (AIDente v1.24 and its assistant tool as well) using an uninstaller. However, the laptop continues to shut down. I'm planning to reinstall the operating system, hoping it will help...
Could you please tell me what else I can do?
Thank you in advance for your response!

**Comment by MatthiasKerbl** [comment]
Hi @AndriiAlerhush,

It sounds like you have a severe case of battery miscalibration.
First, to avoid this in the future, please use the Calibration Mode of AlDente Pro every two weeks or fully charge and discharge your battery at least every two weeks. Furthermore, in such a severe case of miscalibration, AlDentes Calibration Mode does not help currently. It prevents the battery from becoming miscalibrated.
To get your battery out of this state, try the following:

1. Charge your battery to 100% and leave it plugged in for 1h+ after it reached 100%.
2. Unplug your MacBook and let it discharge as slowly as possible, the best is to just leave it idle and avoid letting it go to sleep.
3. When the MacBook shuts down again or the battery is empty, repeat steps 1 and 2 as often as necessary to calibrate the battery again and it normally discharges to <5% again. Sometimes, this can take 5+ cycles.

Best,
Matthias

**Reply by AndriiAlerhush** [reply]
Hi,
After it turns off, if I connect the power cable and restart it, then it shows over 40%. Would reinstalling the operating system not help? I actively use the laptop and don't have much time for these manipulations. Would it be acceptable to discharge it by programming on it? Can this situation significantly affect the battery's health? In the settings, it shows a healthy 100%. It is now charging the battery with very high power.
Thank you for your response!

<img width="371" alt="Screenshot 2024-03-27 at 11 08 08" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/163200152/6f86127c-9c5d-4e52-b059-bb131f197365">

**Reply by MatthiasKerbl** [reply]
Hi @AndriiAlerhush,

> Would reinstalling the operating system not help?

Not to my knowledge, you need to calibrate your battery.

> Would it be acceptable to discharge it by programming on it?

Yes, just try to avoid energy intense tasks.

> Can this situation significantly affect the battery's health?

Not to my knowledge.

Also, here is the answer from our FAQ about this problem:

_Keeping your battery at a lower percentage, such as under 80%, over weeks without doing full cycles (100%-0%-100%) can result in a disturbed battery calibration. When this happens, your Macbook might turn off with 40-50% left or it will stay at 100% for hours, and your battery capacity can drop significantly. However, this is only due to a disturbed battery calibration and not because of a faulty or degraded battery. To avoid this issue, we recommend doing at least one full cycle (0%-100%) every two weeks. Even if your battery calibration gets disturbed, doing 4-5 full cycles will recalibrate your battery and the capacity will go up again. AlDente Pro has a feature called Calibration Mode(https://apphousekitchen.com/feature-explanation-calibration-mode-2/), which will automatically do a full cycle when started. You can read more about the calibration of “smart” batteries here: https://batteryuniversity.com/article/bu-603-how-to-calibrate-a-smart-battery_

Best,
Matthias
